cli/tauri-bundler/src/bundle/common.rs

```diff
@@ -277,7 +277,7 @@ mod tests {
         create_file(&tmp.path().join("orig/sub/file.txt")).expect("Unable to create file");
       writeln!(file, "Hello, world!").expect("Unable to write to file");
     }
-    symlink_file(
+    super::symlink_file(
```

Comment:
Why move this out of the main import statement?

Reply:
because it doesn't run on Windows, there's something wrong happening and i'll take a look. (and unused imports are a clippy warning).

Reply:
Ahh that part is windows only, my bad.